module.css

```diff
@@ -28,6 +28,8 @@
 .pb3 { padding-bottom: var(--space-3) }
 .pb4 { padding-bottom: var(--space-4) }
 
+.-m2 { margin: calc(var(--space-2) * -1) }
+.-m1 { margin: calc(var(--space-1) * -1) }
```

Comment:
[Dash is allowed albeit often for vendor stuff but this seems safe enough.](https://stackoverflow.com/a/449000/770127)